src/locale.ts

```diff
@@ -85,6 +86,7 @@ export {
   thTH,
   trTR,
   ukUA,
+  useLocaleReceiver,
```

Comment:
i get that you like things alphabetized, but this mixed in with all the locales feels a little off